client/src/components/Tabs/TabItem.js

```diff
@@ -1,62 +1,22 @@
-import React, { Component } from 'react';
-import { Fade, TabPane } from 'reactstrap';
+import React from 'react';
+import { TabPane } from 'reactstrap';
 import PropTypes from 'prop-types';
 
-class TabItem extends Component {
-  /**
-   * Fetches the properties for the tab content
-   *
-   * @returns {object} properties
-   */
-  getTabProps() {
-    const {
-      name,
-      className,
-      extraClass,
-      disabled,
-      } = this.props;
-
-    return {
-      tabId: name,
-      className: `${className} ${extraClass}`,
-      disabled,
-    };
-  }
-
-  /**
-   * Checks if the tab is the active tab
-   *
-   * @returns {bool} active
-   */
-  isActive() {
-    return this.context.activeTabId === this.props.name;
-  }
-
-  render() {
-    const tabProps = this.getTabProps();
-    return (
-      <TabPane {...tabProps}>
-        <Fade in={this.isActive()}>
-          {this.props.children}
-        </Fade>
-      </TabPane>
-    );
-  }
-}
+const TabItem = ({ name, className, extraClass, disabled, children }) => (
```

Comment:
I've reformatted this component to remove the transition because it's buggy on Reactstrap 8. BUt PUX needs to sign off on this before we merge.

Reply:
Sacha has provided the thumbs up of approval.